Gives deuterium and tritium independent mass and capacity

l3lb0t opened the pull request:
### Description
As stated above, this pull request gives deuterium and tritium gas their own mass and heat capacity because I felt that it didn't make sense that they inherited their mass and heat capacity from hydrogen. Admittedly, this is based on my own (limited) research, so the figures or concept may not even be correct.

🆑  karljohansson
tweak: Deuterium gas has a molar mass of 0.004
tweak: Deuterium gas has a heat capacity of 80
tweak: Tritium gas has a molar mass of 0.006
tweak: Tritium gas has a heat capacity of 60
tweak: The R-UST engine is slightly less energy efficient, more so using tritium based setups
/:cl:

SierraKomodo reviewed (dismissed):
Changelog isn't formatted correctly for the bot to parse it. See https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation

SierraKomodo commented on code/modules/xgm/gases.dm:
```diff
@@ -113,12 +113,16 @@
 /decl/xgm_gas/hydrogen/deuterium
 	id = GAS_DEUTERIUM
 	name = "Deuterium"
+	specific_heat = 80	// J/(mol*K)
+	molar_mass = 0.004	// kg/mol
```
Comments on the overrides aren't really necessary or helpful. If you did want to document the intended measurement unit, add it as a [dmdoc](https://github.com/Baystation12/Baystation12/wiki/Guide-to-SpacemanDMM-Docblocks) to the original `var/specific_heat` and `var/molar_mass` definitions for `/decl/xgm_gas`.

```suggestion
	specific_heat = 80
	molar_mass = 0.004
```

SierraKomodo commented on code/modules/xgm/gases.dm:
```diff
@@ -113,12 +113,16 @@
 /decl/xgm_gas/hydrogen/tritium
 	id = GAS_TRITIUM
 	name = "Tritium"
+	molar_mass = 0.006	// kg/mol
+	specific_heat = 60	// J/(mol*K)
```
```suggestion
	molar_mass = 0.006
	specific_heat = 60
```

SierraKomodo reviewed (commented):
Note for myself and other reviewers - Test to verify this doesn't do weird things to RUST setups and maybe consider relevant wiki updates (Assuming the rust guide on the wiki is even up to date right now anyway).

l3lb0t commented:
> Note for myself and other reviewers - Test to verify this doesn't do weird things to RUST setups and maybe consider relevant wiki updates (Assuming the rust guide on the wiki is even up to date right now anyway).

Just checked, the R-UST appears to work fine on an all deuterium setup with hydrogen injecting at 10kpa

SierraKomodo reviewed (changes requested):
Final thing - The two new commits need to be squashed into the prior two commits, or just squash them altogether. Clean commit history without a bunch of addendums in a PR and all that.

l3lb0t commented:
> Final thing - The two new commits need to be squashed into the prior two commits, or just squash them altogether. Clean commit history without a bunch of addendums in a PR and all that.

Well, I tried. I now have six commits.

SierraKomodo commented:
https://github.com/Baystation12/Baystation12/wiki/Squashing might be helpful. If you still need help with squashing, pop into the `#code` channel on our discord server: https://bay.ss13.me/discord

l3lb0t committed:
Gives Deuterium and Tritium Gas Independent Mass and Heat Capacity

This commit makes deuterium and tritium heavier than hydrogen, in order to be more aligned with reality.

l3lb0t commented:
> Final thing - The two new commits need to be squashed into the prior two commits, or just squash them altogether. Clean commit history without a bunch of addendums in a PR and all that.

Done. I'd like to thank Baystation discord member Domic for being incredibly patient and helping me fix the issue.
